mlir/lib/CAPI/Dialect/Rock.cpp

```diff
@@ -138,3 +141,69 @@ MLIR_CAPI_EXPORTED size_t mlirRockTuningGetKey(MlirModule module, char *buf,
   strncpy(buf, perfStr.c_str(), bufLen);
   return perfStr.size();
 }
+
+// TODO (ravil)
+MLIR_CAPI_EXPORTED
+enum RocmlirSplitKSelectionLikelihood
+mlirIsSplitKFaster(int64_t gDim, int64_t mDim, int64_t nDim, int64_t kDim,
+                   int64_t numCUs, RocmlirTuningParamSetKind tuningLevel) {
+  return rock::isSplitKFaster(gDim, mDim, nDim, kDim, numCUs);
+}
+
+// TODO (ravil): document
+MLIR_CAPI_EXPORTED
+bool mlirIsModuleFusible(MlirModule module, MlirStringRef perfStr) {
+  auto mod = unwrap(module);
+  StringRef perfConfig = unwrap(perfStr);
+
+  if (!rock::isSplitKRequested(mod, perfConfig)) {
+    return true;
+  }
+  return succeeded(rock::testFusibility(mod));
+}
+
+// TODO (ravil): document
+MLIR_CAPI_EXPORTED
+size_t mlirGetNumPrefillArgs(MlirModule module) {
+  auto mod = unwrap(module);
+  assert(mod.getRegion().getBlocks().size() == 1 &&
+         "expected a single block/function in a module");
+
+  std::optional<LLVM::LLVMFuncOp> func = std::nullopt;
+  mod.walk([&](LLVM::LLVMFuncOp op) { func = op; });
```

Comment:
... having looked at documentation some more, there're probably places where we called `walk()` where we just wanted https://mlir.llvm.org/docs/Tutorials/UnderstandingTheIRStructure/#filtered-iterator-getopsopty

Reply:
In theory, `getOps<LLVM::LLVMFuncOp> should work but for some reason it returns `nullptr` for me